neurograph.py

```diff
@@ -0,0 +1,170 @@
+import networkx as nx
+from networkx.algorithms import bipartite
+
+#input: DAG g as an adjacent list representation 
```

Comment:
Good to have these as doc-strings in the method: https://www.programiz.com/python-programming/docstrings

Reply:
Resolved